extensions/wikia/CommunityPage/scripts/ext.communityPage.js

```diff
@@ -251,6 +251,15 @@ require([
 				handleClick(event, category);
 			}
 		});
+
+		// Track clicks in the To-do List module
+		$('.community-page-todo-list-module-edit').on('mousedown touchstart', 'a', function (event) {
+			handleClick(event, 'community-page-todo-list-module-edit');
+		});
+
+		$('.community-page-todo-list-module-content').on('mousedown touchstart', 'a', function (event) {
+			handleClick(event, 'community-page-todo-list-content');
```

Comment:
I'd consider unifying these two tracking events into one, similar to what we do elsewhere in this code and use either a label or class to determine the tracking label.